apps/dashboard/src/components/subscribers/subscriber-overview-form.tsx

```diff
@@ -75,8 +79,9 @@ export function SubscriberOverviewForm(props: SubscriberOverviewFormProps) {
 
   const { patchSubscriber } = usePatchSubscriber({
     onSuccess: (data) => {
-      showSuccessToast(`Updated subscriber: ${getSubscriberTitle(subscriber)}`, undefined, toastOptions);
+      showSuccessToast(`Updated subscriber: ${getSubscriberTitle(data)}`, undefined, toastOptions);
```

Comment:
fix for the latest subscriber data, otherwise old name and surname will be shown